README.md

```diff
@@ -3,49 +3,96 @@
-| Form App | ![Deploy Form App to GitHub Pages](https://github.com/IEA-Task-43/digital_wra_data_standard/workflows/Deploy%20Form%20App%20to%20GitHub%20Pages/badge.svg) | [Form App](https://iea-task-43.github.io/digital_wra_data_standard/) |
+To learn more about the _WRA Data Model_, please read Amit Bohara's 
+[introduction](https://github.com/IEA-Task-43/digital_wra_data_standard/wiki/Task-43-WRA-Data-Model---An-introduction)
+in the Wiki section of this GitHub repository.
 
-## Digital Calibration Certificate Model
+JSON Schema is used to capture the _WRA Data Model_. It can be thought of as a blueprint for how an implementation of the
+data model can be described with the JSON data format. See Figure 1 below for a snippet of an example implementation.
 
-As a requirement resulting from the development of the *WRA Data Model*, a [digital calibration certificate](./digital_calibration_certificate) is introduced as a complementary standard. This standard currently supports anemometer calibration certificates according to IEC 61400-12-1:2017. 
+![example_implementation](https://user-images.githubusercontent.com/25622575/211047742-e83ee47b-d756-4e5e-a48f-cfb3d2fa00c6.png)
+<br>
+_Figure 1: Example implementation of the WRA Data Model._
 
-## Contributing to the Standard
-We welcome all contributions including issue reporting, new features and bug fixes.   For full details see the contributing guidelines and meeting notes here:
+The following tools are part of this undertaking:
 
-- [Contribution Guidelines](./contributing.md)
-- [Meeting Notes](https://github.com/IEA-Task-43/digital_wra_data_standard/discussions/129#discussion-3748501)
+- [WRA Data Model](./schema/iea43_wra_data_model.schema.json): This JSON Schema file is the _WRA Data Model_. It describes 
+  how a JSON file that describes wind resource measurement data should be composed.
+
+- [Documentation](./docs/README.md): Markdown documentation for the _WRA Data Model_ created directly from the JSON Schema.
+
+- [Form App](https://iea-task-43.github.io/digital_wra_data_standard/): This app shows a form that is modeled after the 
+  JSON Schema and can create JSON data out of your inputs that is in accordance with the _WRA Data Model_.
+
+- [Python Data Model Loading Example](./tools/load_demo_schema.ipynb): This notebook shows how to read an example file that 
+  uses the _WRA Data Model_ with Python. [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/IEA-Task-43/digital_wra_data_standard/master?filepath=.%2Ftools%2Fload_demo_schema.ipynb)
 
 ## Dissemination
-The Task 43 team is actively working to disseminate information and improve user experience.  See below:
+The Task 43 team is actively working to disseminate information and improve user experience. 
+The [Wiki](https://github.com/IEA-Task-43/digital_wra_data_standard/wiki) tab in this GitHub repository is the starting 
+point to learn more via tutorials, recorded presentations and more.
+
```

Comment:
suggest including some of the resources previously generated (at least videos):
- [Presentations](https://github.com/IEA-Task-43/digital_wra_data_standard/wiki/Presentations)-Maybe this should be deprecated?
- [Videos](https://www.youtube.com/watch?v=BehksiZrvnw)

Reply:
I don't think the 'Presentation' should be deprecated. In fact those presentations ARE the videos.

I was thinking that we just get them to the Wiki where everything will be. The Wiki page needs serious improvement.

I have put the wiki link as a bullet too to make it more obvious that this is a link to the place where you can find all dissemination info.